specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/keys.json

```diff
@@ -1356,8 +1279,9 @@
           "description": "True if the key's lifetime is managed by key vault. If this is a key backing a certificate, then managed will be true."
         },
         "release_policy": {
-          "$ref": "#/definitions/KeyReleasePolicy",
-          "description": "The policy rules under which the key can be exported."
+          "description": "Blob encoding the policy rules under which the key can be exported.",
```

Comment:
Should be more specific about supported encodings since I don't see a `contentType` property like we had discussed over Teams. I.e. either say something like, "Base64-encoded UTF-8 string encoding the policy rules under which the key can be exported." Maybe not the best, but hopefully conveys my concern.

Still, given conversations about versioning, I expected to see a `contentType` that describes what is in this. Values could look like:

* `application/json`
* `application/json; charset=utf-8`

Either now or in the future, you may also need a `version` property. While HTTP Content-Type headers don't allow it, you're in control of this property so you could just append something like `; version=2` to the `contentType`, or just introduce a new property sibling to `contentType`. Seems better to define that all now so we have the proper support *and defaults* known and documented now.